org-super-agenda.el

```diff
@@ -272,12 +272,15 @@ face `org-super-agenda-header' appended, and the text properties
                           (string org-super-agenda-header-separator))))
          (setq s (concat " " s))
          (add-face-text-property 0 (length s) 'org-super-agenda-header t s)
-         (org-add-props s nil
+         (org-add-props s nil 'face 'org-super-agenda-header
            'keymap org-super-agenda-header-map
            ;; NOTE: According to the manual, only `keymap' should be necessary, but in my
            ;; testing, it only takes effect in Agenda buffers when `local-map' is set, so
            ;; we'll use both.
-           'local-map org-super-agenda-header-map)
+           'local-map org-super-agenda-header-map
+           'org-super-agenda-header t)
+         (org-add-props org-super-agenda-header-separator nil
+           'org-super-agenda-header t)
          ;; Don't apply faces and properties to the separator part of the string.
          (concat separator s)))))
```

Comment:
Maybe, instead of using `org-add-props` on `separator` and `s`, we should simply use `propertize` on this call to `concat`.  Or would the keymaps be a problem?  What do you think?

Reply:
I can still call any org-agenda-mode-map bindings on the seperator.